The headers are generated by the rpcgen just built, which is a program of the target in a cross
build: RPCGEN_RUN puts a runner in front of it (bbxb: <HARCH>-run).

--- a/rpcsvc/Makefile.am
+++ b/rpcsvc/Makefile.am
@@ -11,5 +11,7 @@
 	bootparam_prot.h mount.h rex.h rusers.h key_prot.h \
 	nfs_prot.h rquota.h sm_inter.h
 
+RPCGEN_RUN =
+
 .x.h:
-	$(top_builddir)/rpcgen/rpcgen -h -o $@ $<
+	$(RPCGEN_RUN) $(top_builddir)/rpcgen/rpcgen -h -o $@ $<
